.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -355,21 +367,41 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        running_mode: ["classic-interp", "fast-interp", "jit", "aot"]
+        running_mode:
+          ["classic-interp", "fast-interp", "jit", "aot", "fast-jit"]
         test_option:
           [
             $DEFAULT_TEST_OPTIONS,
             $MULTI_MODULES_TEST_OPTIONS,
             $SIMD_TEST_OPTIONS,
             $THREADS_TEST_OPTIONS,
           ]
+        exclude:
+          # uncompatiable mode and feature
+          # classic-interp and fast-interp don't support simd
+          - running_mode: "classic-interp"
+            test_option: $SIMD_TEST_OPTIONS
+          - running_mode: "fast-interp"
+            test_option: $SIMD_TEST_OPTIONS
+          # aot and jit doesn't support multi module
+          - running_mode: "aot"
+            test_option: $MULTI_MODULES_TEST_OPTIONS
+          - running_mode: "jit"
+            test_option: $MULTI_MODULES_TEST_OPTIONS
+          # fast-jit only tested on default mode, exclude other three
+          - running_mode: "fast-jit"
+            test_option: $MULTI_MODULES_TEST_OPTIONS
+          - running_mode: "fast-jit"
+            test_option: $SIMD_TEST_OPTIONS
+          - running_mode: "fast-jit"
+            test_option: $THREADS_TEST_OPTIONS
     steps:
       - name: checkout
         uses: actions/checkout@v3
 
       #only download llvm libraries in jit and aot mode
       - name: Get LLVM libraries
-        if: matrix.running_mode == 'jit' || matrix.running_mode == 'aot'
+        if: matrix.running_mode != 'classic-interp' && matrix.running_mode != 'fast-interp'
```

Comment:
maybe we shall add a step to prepare variables to control llvm library downloading and x86-32 test. In order to write those long `if` conditions fewer times.